src/assets/custom/js/accordion.js

```diff
@@ -0,0 +1,74 @@
+(function() {
+
+  var ACTIVE_CLASS = "active";
+
+  var SELECTORS = {
+    ACCORDION: "[data-accordion]",
+    PANEL: "[data-accordion_panel]",
+    PANEL_ACTIVE: "[data-accordion_panel]." + ACTIVE_CLASS
```

Comment:
Is it possible to use template literals/strings here? :)

Reply:
Maybe! Maybe @Dan-Bird can have a look at this

Reply:
Ya, no worries, I've approved it anyways :)